datafusion/core/src/sql/planner.rs

```diff
@@ -293,6 +294,8 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
 
         let plan = self.order_by(plan, query.order_by)?;
 
+        let plan: LogicalPlan = self.offset(plan, query.offset)?;
```

Comment:
Could you add some tests demonstrating SQL queries with the various combinations of `OFFSET` and `LIMIT`